bed/compare.go

```diff
@@ -160,6 +161,27 @@ func CompareChromEndByChrom(a Bed, b Bed) int {
 	return CompareChromEnd(a, b)
 }
 
+//CompareDistance compares beds by chromStart and chromEnd and returns the minimum distance between
+//the two beds. return options; -1 (different chromosomes, no distance calculated),
+//0 (overlap, minimum distance is 0), >=0 is the minimum distance.
+func CompareDistance(a Bed, b Bed) int {
+	chromComp := strings.Compare(a.Chrom, b.Chrom)
+	if chromComp == 0 {
```

Comment:
I think this should be `chromComp != 0`.

Reply:
Since you are not ordering the chroms, but just testing for equality, you can also do "if a.Chrom != b.Chrom {"